SCons/Tool/ninja/__init__.py

```diff
@@ -186,7 +186,7 @@ def generate(env):
     env.Append(BUILDERS={"Ninja": ninja_builder_obj})
 
     env["NINJA_ALIAS_NAME"] = env.get("NINJA_ALIAS_NAME", "generate-ninja")
-    env['NINJA_DIR'] = env.get("NINJA_DIR", env.Dir(".ninja").path)
+    env['NINJA_DIR'] = env.get("NINJA_DIR", ".ninja")
```

Comment:
making it a dir causes scons to create a node which then attempts to create the directory for ninja, but ninja will already do this, so it ends up with an unnecessary warning message that a directory already exists.

Reply:
Should this instead be `#/.ninja` ?